wpeverest/everest-forms
# Enhancement/multiple choices preview

### All Submissions:






### Changes proposed in this Pull Request:

 - This PR enables the preview feature to show the amount regardless of payment currency in the preview section while updating the values.

### How to test the changes in this Pull Request:

1. Go to Payment Field while adding multiple choices.
2. Type the amount for the field which reflects in preview section.
3. Save the changes to see into final output.

### Types of changes:




### Other information:




### Changelog entry

> Added preview for amount while changing into the payment field for multiple choices and checkboxes.
